stern/tail.go

```diff
@@ -99,6 +99,25 @@ func (o TailOptions) IsInclude(msg string) bool {
 	return false
 }
 
+var colorHighlight = color.New(color.FgRed, color.Bold).SprintFunc()
+
+func (o TailOptions) HighlightMatchedString(msg string) string {
+	matched := false
+
+	for _, rin := range o.Include {
+		msg = rin.ReplaceAllStringFunc(msg, func(part string) string {
+			matched = true
+			return colorHighlight(part)
+		})
+
+		if matched {
+			break
+		}
```

Comment:
We can specify the include option multiple times, but it seems that this logic highlights first matching regular expression only. Was this done intentionally?

<img width="932" alt="image" src="https://user-images.githubusercontent.com/9250296/219847657-9e93cdb2-577c-4976-9aaf-58b57ababbe4.png">

Reply:
Oh, I thought I should only highlight the first match, but I was wrong. I will fix it.

Reply:
I've fixed it in 06904a6.

![image](https://user-images.githubusercontent.com/230185/219866880-bbb1bbdc-ccb8-421f-ab2b-4117904ec679.png)